board/safety.h

```diff
@@ -249,6 +249,10 @@ int to_signed(int d, int bits) {
   return d_signed;
 }
 
+bool get_param_flag(int16_t param, uint16_t mask) {
```

Comment:
I don't think this really needs a function, but if we keep it, it should probably be more generic like `get_flag` so it can be used for the unsafe modes too.

Reply:
due to misra checks this requires a cast from `int16_t` to `uint16_t` (see latest commit) and the fact that it uses int16 in general makes it somewhat specific to safety mode param.  Should I move it and re-name it to `get_int16_flag()` or something?

Reply:
hmm, then maybe just make it a macro?

Reply:
good idea, the macro can abstract away the type